CONTRIBUTING.md

```diff
@@ -47,47 +48,42 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) is provided as a lightweight, convenient testbed to try
+your changes to the extension.
 
-- Go to the `demo/` directory of this repo.
+> **Note**: It is not an exhaustive showcase of DVC's features. Testers are
+> encouraged to try other DVC repositories -- especially real-world cases!
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Run `yarn setup:venv` from the root of this repo to set up a virtual Python
+  environment for the demo project (in `demo/.env`).
 
-- `yarn install` is a prerequisite for automatically setting up the demo
-  project.
+- Open the `demo/` folder of this repo directly in VS Code.
 
-- Run `yarn setup:venv` from the monorepo root (not `./demo`) to automatically
-  setup the Python virtual environment in the demo project.
+- Pull the project data with the extension (either from the **DVC Tracked**
```

Comment:
`with the extension` -> using the extension ...

Reply:
```suggestion
- Pull the project data using the extension (either from the **DVC Tracked**
```